hkmc2/shared/src/test/mlscript-compile/Block.mls

```diff
@@ -29,7 +29,7 @@ class Case with
   constructor
     Lit(val lit: Literal)
     Cls(val cls: Symbol, val path: Path)
-    Tup(val len: Int) // TODO: remove inf?
```

Comment:
FYI `inf = true` is suppossed to indicate that the `len` is a lower bound and that the tuple could be bigger. It's what you get from a pattern like `[0, 1, ...xs, 8, 9]`.